server/package.json

```diff
@@ -14,14 +14,15 @@
     "postinstall": "npm run build"
   },
   "dependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
```

Comment:
See the above comment about `"useBuiltIns"`. I think if we use it, we need to include the `@babel/polyfill` dependency too? I wasn't 100% sure about this one, so let me know if you have input on this.